package.json

```diff
@@ -22,10 +22,10 @@
     "peerDependencies": {
         "react": "^16.8.0",
         "react-dom": "^16.8.0",
-        "@material-ui/core": "^3.2.0"
+        "@material-ui/core": "^4.0.0-beta.0"
```

Comment:
4.0.0 has been released to stable. This should at least be changed to `^4` however if this lib doesn't include any v4 exclusive features (which I think is the case) then I strongly recommend changing it to `>=3 <= 5` which will allow any version higher than 3.0.0 and lower than 5 (whenever that releases, and this being a safety net in case 5 breaks the lib).

```json5
// other stuff
  "peerDependencies": {
        "react": ">=16.8.0 <= 17",
        "react-dom": ">=16.8.0 <= 17",
        "@material-ui/core": ">=3 <= 5"
}

// more stuff
```

Also gave React the same treatment for the same reasons.

Reply:
3.0 would require use of `RootRef`. We could support both versions but then this library would trigger warnings in StrictMode and would ultimately be not viable for concurrent react.

Reply:
Fair enough, then I would suggest doing `^4` as version for the peer dependency.